backend/sumo-championship/src/main/java/com/sumoc/sumochampionship/db/people/OwnedClubId.java

```diff
@@ -0,0 +1,14 @@
+package com.sumoc.sumochampionship.db.people;
+
+import java.io.Serializable;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class OwnedClubId implements Serializable {
```

Comment:
Club has got attribute "trainers". It stores all WebsiteUser that can manage certain club. Please use it instead of creating new Entity. New Entity will cause the integration problem (we are using club.trainers in the code but now it will contains none relations)